cmd/src/debug_comp.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+	"unicode"
+
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+func init() {
+	usage := `
+'src debug comp' mocks docker cli commands to gather information about a docker-compose Sourcegraph instance.
+
+Usage:
+
+    src debug comp [command options]
+
+Flags:
+
+	-o			Specify the name of the output zip archive.
+	-cfg		Include Sourcegraph configuration json. Defaults to true.
+
+Examples:
+
+    $ src debug comp -o debug.zip
+
+	$ src -v debug comp -cfg=false -o foo.zip
+
+`
+
+	flagSet := flag.NewFlagSet("comp", flag.ExitOnError)
+	var base string
+	var configs bool
+	flagSet.BoolVar(&configs, "cfg", true, "If true include Sourcegraph configuration files. Default value true.")
+	flagSet.StringVar(&base, "o", "debug.zip", "The name of the output zip archive")
+
+	handler := func(args []string) error {
+		if err := flagSet.Parse(args); err != nil {
+			return err
+		}
+
+		//validate out flag
+		if base == "" {
+			return fmt.Errorf("empty -o flag")
+		}
+		// declare basedir for archive file structure
+		var baseDir string
+		if strings.HasSuffix(base, ".zip") == false {
```

Comment:
```suggestion
		if !strings.HasSuffix(base, ".zip") {
```

Try not to compare explicitly against booleans

Reply:
fixed